Riot/Modules/PhotoPicker/PhotoPickerPresenter.swift

```diff
@@ -0,0 +1,107 @@
+// 
+// Copyright 2022 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import UIKit
+import PhotosUI
+import CommonKit
+
+@available(iOS 14.0, *)
+protocol PhotoPickerPresenterDelegate: AnyObject {
+    func photoPickerPresenter(_ presenter: PhotoPickerPresenter, didPickImage image: UIImage)
+    func photoPickerPresenterDidCancel(_ presenter: PhotoPickerPresenter)
+}
+
+/// A picker for photos and videos from the user's photo library on iOS 14+ using the
+/// new `PHPickerViewController` that doesn't require permission to be granted.
+@available(iOS 14.0, *)
+final class PhotoPickerPresenter: NSObject {
```

Comment:
Makes sense, by the time we're iOS 14+ this will hopefully be able to replace MediaPickerViewController anyway so I don't think the name clash is worth worrying about too much 👍

Reply:
I've named the module directory `MediaPickerV2` so to keep it separate from the other files.